devops/consuming_conan_center.rst

```diff
@@ -0,0 +1,71 @@
+
+Most of these reasons are not unique to Conan, but to every other package manager for other programming languages. 
+
+.. important::
+  
+  The general devops known good practice is that you shouldn't rely on central repositories on production systems, but do consider the following approaches:
+
+  - If your project is mostly a "consuming" project, i.e. you are not generating your own packages, but just consuming packages from ConanCenter, and the above issues are not a concern because your project does not need that kind of robustness or security, then you can use **lockfiles** to at least guarantee that your project build using the same dependencies over time. Check :ref:`the lockfile docs<tutorial_versioning_lockfiles>` for more information.
+  - If you are creating your own packages for your own libraries or applications, or any of the above reasons is a concern for your production environment, then the recommended approach is **hosting your own copy of the packages you need for production** in your own private server.
+
+We have seen many Conan users succesfully using the second approach, building and hosting their own binaries in their own repository, even for "consuming" cases. The process for doing that is not complicated (and we are working to further streamline it) and in practice it is advangeous in most cases. Let's see how this can be done:
+
+
+
+Creating and hosting your own ConanCenter binaries
+--------------------------------------------------
+
+Hosting your own copy of the packages you need in your server could be done by just downloading binaries from ConanCenter and then uploading them to your own server. However, it is much better to fully own the complete supply chain and create the binaries in your own CI systems. So the recommended flow to use ConanCenter packages in production would be:
+
+- Create a fork of the ConanCenter Github repository: https://github.com/conan-io/conan-center-index
+- Create a list of the packages and versions you need for your projects. This list can be added to the fork too, and maintained there (packages can be added and removed with PRs when the teams need them).
+- Create a script that first ``conan export`` all the packages in your list, then ``conan create --build=missing`` them. Do not add ``user/channel`` to these packages, it is way simpler to use them as ``zlib/1.2.13`` without user-channel. The ``user/channel`` part would be mostly recommended for your own proprietary packages, but not for open source ConanCenter packages.
+- Upload your build packages to your own server, that you use in production, instead of ConanCenter.
+
+This is the basic flow idea. We will be adding examples and tools to further automate this flow as soon as possible.
```

Comment:
It would be amazing to have blank pages as place holders so people searching for this can find them

Just link back here in the meantime so the information is surfaced 🙏 

The docs search matches page titles and so it would be easier to find